content/cli/kes-ls/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: kes ls
+date: 2023-03-03
+lastmod: :git
+draft: false
+tableOfContents: true
+---
+
+{{< admonition title="Replaces deprecated commands" type="note" >}}
```

Comment:
I know the subject for this phrase is in the immediately following line, but grammatically it's making me go 🤔
Maybe?
```suggestion
{{< admonition title="`ls` replaces deprecated commands" type="note" >}}
```
(if that even works)

Reply:
Well, I can't get the command to take formatting. So I'm just dropping the admonition title.